DemoViper/Sources/Modules/ListRepos/View/ReposViewController.swift

```diff
@@ -80,6 +80,7 @@ extension ReposViewController {
         setupNavigationBar()
         setupTableView()
         setupEmptyView()
```

Comment:
Please update `setUp` for these functions

Reply:
I renamed those functions in PR #15. Please check it out.